projects/plugins/crm/.gitattributes

```diff
@@ -1,24 +1,7 @@
-# Files not needed to be distributed in the package.
-.gitattributes       export-ignore
-.github/             export-ignore
-package.json         export-ignore
-gulpfile.esm.js      export-ignore
-.env                 export-ignore
-.env.testing         export-ignore
-codeception.yml      export-ignore
-codeception.dist.yml export-ignore
-.wp-env.json         export-ignore
-/.wordpress-org/**/* export-ignore
-/.wordpress-org      export-ignore
-.wordpress-org/**/*  export-ignore
-.wordpress-org       export-ignore
-/node_modules/**/*   export-ignore
-/node_modules/*      export-ignore
-node_modules         export-ignore
-
 # Files to include in the mirror repo, but excluded via gitignore
 # Remember to end all directories with `/**` to properly tag every file.
 # /src/js/example.min.js  production-include
+build/**                                        production-include
 jetpack_vendor/**                               production-include
```

Comment:
Is this one we need in production?

Reply:
It is: https://github.com/Automattic/jetpack/pull/30058#discussion_r1166544608